discovery-provider/src/api/v1/users.py

```diff
@@ -797,6 +806,74 @@ def get(self, id: str, authed_user_id: Optional[int] = None):
         return success_response(tracks)
 
 
+def get_user_collections(
+    id: str, collection_type: CollectionType, authed_user_id: Optional[int] = None
+):
+    """Fetches albums or playlists from a user's library"""
+    args = user_collections_library_parser.parse_args()
+    decoded_id = decode_with_abort(id, ns)
+    if authed_user_id != decoded_id:
```

Comment:
cc: @nicoback2 I think we're going to do away with this check in the future, and instead have the auth decorator handle it so we can support delegation etc. But this is fine for now

Reply:
yes sg